nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/state/StandardStateManager.java

```diff
@@ -24,21 +24,34 @@
 import org.apache.nifi.logging.ComponentLog;
 import org.apache.nifi.logging.LogRepository;
 import org.apache.nifi.logging.LogRepositoryFactory;
-import org.apache.nifi.processor.SimpleProcessLogger;
 import org.apache.nifi.logging.StandardLoggingContext;
+import org.apache.nifi.processor.SimpleProcessLogger;
 
 import java.io.IOException;
 import java.util.Map;
+import java.util.function.Supplier;
 
 public class StandardStateManager implements StateManager {
     private final StateProvider localProvider;
     private final StateProvider clusterProvider;
     private final String componentId;
+    private final Supplier<Boolean> dropStateKeySupportedSupplier;
 
     public StandardStateManager(final StateProvider localProvider, final StateProvider clusterProvider, final String componentId) {
+        this(localProvider, clusterProvider, componentId, () -> false);
+    }
+
+    public StandardStateManager(final StateProvider localProvider, final StateProvider clusterProvider,
```

Comment:
Is this constructor needed in addition to the `Supplier` based constructor?

Reply:
No longer needed, also a leftover. Removing.